README.md

```diff
@@ -270,6 +328,56 @@ export function parse(
 }
 ```
 
+## MessageBuilder
+
+`zod-validation-error` can be configured with a custom `MessageBuilder` function in order to produce case-specific error messages.
+
+#### Example
+
+For instance, one might want to print the error messages to the console and color any `invalid_string` error with yellow.
+
+```typescript
+import { z as zod } from 'zod';
+import { type MessageBuilder, fromError } from 'zod-validation-error';
+import chalk from 'chalk';
+
+// create custom MessageBuilder
+const myMessageBuilder: MessageBuilder = (issues) => {
+  return issues
+    // format error message
+    .map((issue) => {
+      if (issue.code === zod.ZodIssueCode.invalid_string) {
+        return chalk.yellow(issue.message);
+      }
+
+      return issue.message;
+    })
+    // join as string with new-line character
+    .join('\n');
+}
+
+// create zod schema
+const zodSchema = zod.object({
+  id: zod.number().int().positive(),
+  email: zod.string().email(),
+});
+
+// parse some invalid value
+try {
+  zodSchema.parse({
+    id: 1,
+    email: 'foobar', // note: invalid string
+  });
+} catch (err) {
+  const validationError = fromError(err, {
+    messageBuilder: myMessageBuilder
+  });
+  // the error is now displayed with red letters
```

Comment:
[nit] According to `L350` I think the error will be displayed with yellow letters 😅

Reply:
Bloody hell.